metadata.rb

```diff
@@ -6,9 +6,9 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 source_url 'https://github.com/dhoer/chef-nssm'
 issues_url 'https://github.com/dhoer/chef-nssm/issues'
-version '3.0.0'
+version '4.0.0'
 
-chef_version '>= 12.6'
+chef_version '>= 12.7'
 
 supports 'windows'
-depends 'windows'
+depends 'windows', '~> 3.1'
```

Comment:
Can this version restriction be left out?  I rather have the user decide what version of windows.